storage/s3/s3.go

```diff
@@ -44,6 +46,8 @@ func New(opts *Options) (storage.Storage, error) {
 	var creds *credentials.Credentials
 	if len(opts.Access) != 0 && len(opts.Secret) != 0 {
 		creds = credentials.NewStaticV4(opts.Access, opts.Secret, opts.Token)
+	} else if len(opts.FileCredentialsPath) != 0 {
+		creds = credentials.NewFileAWSCredentials(opts.FileCredentialsPath, opts.Profile)
```

Comment:
> If the profile is not set then put it as default.

@donny-dont Just wanted to let you know that `minio` defaults this to default for us (like aws does). 

https://github.com/minio/minio-go/blob/master/pkg/credentials/file_aws_credentials.go#L72:L77